llvm/lib/Transforms/Vectorize/VPlanConstruction.cpp

```diff
@@ -879,52 +867,72 @@ bool VPlanTransforms::handleMaxMinNumReductions(VPlan &Plan) {
   }
 
   VPBasicBlock *LatchVPBB = LoopRegion->getExitingBasicBlock();
+  VPBasicBlock *MiddleVPBB = Plan.getMiddleBlock();
+  VPBuilder MiddleBuilder(MiddleVPBB, MiddleVPBB->begin());
   VPBuilder Builder(LatchVPBB->getTerminator());
```

Comment:
```suggestion
  VPBuilder LatchBuilder(LatchVPBB->getTerminator());
```
consistent with `MiddleBuilder`.